README.md

```diff
@@ -83,16 +75,26 @@ Run tests to verify everything is working.
 
 1. Create a new branch.
 1. Add any new dependencies to any of the packages via `lerna add`.
-1. Run `lerna bootstrap` to make sure everything is installed.
+1. Install all dependencies (see above).
 1. Make your necessary source file changes.
 1. Write your tests if applicable.
 1. Do *NOT* update version numbers in `package.json` files (`lerna publish` does it for you).
 1. Run `yarn run test` to make sure all tests pass.
 1. Commit/push your changes.
 1. Create a pull request against master.
-1. Once your pull request is approved, merge it to master.
-1. Pull latest master locally.
-1. Run `lerna publish` in order to publish your changes to npm.
+1. Once your pull request is approved, run `lerna publish` in order to publish your changes to npm.
+   - Ideally we would first merge the pull request to master, then publish master to npm, but `lerna publish`
+     creates a "Publish" commit which would have to be pushed to master, and non-admins cannot push
+     directly to master.
+   - If this is your first time running `lerna publish`, you will be prompted to first run
+     `npm adduser`. If you don't have an npm account, create one at <https://www.npmjs.com/signup>.
```

Comment:
They'll also need to be added to our organization.

Reply:
Thanks! I've added that detail.